### PR TITLE
[Change] *Review models to be more uniform

### PR DESCRIPTION
**Pull request by adeecc**

### CODE REVIEW

**Review by aryanarora180 — APPROVED** [review]

File: src/controllers/requestReview.ts [hunk 1]
```diff
@@ -72,23 +99,35 @@ export const getRequestReviewsByPostingUser = async (posterId: string) => {
...
-    });
-    await review.save();
-    return {
-      status: true,
-      data: review,
-    };
+    const populatedRequest = await Request.findOne({
+      _id: request,
+    }).exec();
+    console.log(populatedRequest);
```
Comment (rocka0) [inline src/controllers/requestReview.ts:110]: Remove console.log and use logger if needed

File: src/controllers/sellerReview.ts [hunk 2]
```diff
@@ -78,18 +82,19 @@ export const getSellerReviewsByPostingUser = async (user_id: string) => {
...
   comment: string,
   rating: number
 ) => {
   try {
-    const serviceRequest = await Request.findOne({
+    const populatedRequest = await Request.findOne({
       _id: request,
     }).exec();
-    if (serviceRequest?.seller !== null) {
+    console.log(populatedRequest);
```
Comment (rocka0) [inline src/controllers/sellerReview.ts:92]: remove console.log again

**Review by rocka0 — CHANGES_REQUESTED** [review]

File: src/controllers/requestReview.ts (hunk 1 above)
Comment (adeecc) [reply src/controllers/requestReview.ts]: Fixed

File: src/controllers/sellerReview.ts (hunk 2 above)
Comment (adeecc) [reply src/controllers/sellerReview.ts]: Fixed 🌟

**Pull request merged**